### PR TITLE
update/tandem-hide-add-bid-hide-sort TM-1756

### PR DESCRIPTION
**Pull request by scott062**
- Hide sort 
- Hid add to bid list for tandem 2 details page results

### CODE REVIEW

**Comment by scott062** [comment]
@mjoyce91 what is the right way to do prop types/defaults for the `this.props.location.search`?

File: src/Containers/Position/Position.jsx [hunk 1]
```diff
@@ -31,6 +32,13 @@ import {
 } from '../../Constants/PropTypes';
 
 class Position extends Component {
+  getChildContext() {
+    const { tandem } = queryString.parse(this.props.location.search);
```
Comment (mjoyce91) [inline src/Containers/Position/Position.jsx:36]: 👍

File: src/Containers/Position/Position.jsx [hunk 2]
```diff
@@ -129,13 +137,20 @@ Position.contextTypes = {
...
 Position.propTypes = {
   onNavigateTo: PropTypes.func.isRequired,
   match: PropTypes.shape({
     params: PropTypes.shape({
       id: PropTypes.string,
     }),
   }).isRequired,
+  location: PropTypes.shape({
+    search: PropTypes.string,
+  }).isRequired,
```
Comment (mjoyce91) [inline src/Containers/Position/Position.jsx:153]: There might be a default prop constant for this

File: src/Components/PositionTitle/PositionTitle.jsx [hunk 3]
```diff
@@ -128,6 +128,7 @@ class PositionTitle extends Component {
 
 PositionTitle.contextTypes = {
   isClient: PropTypes.bool,
+  isTandemTwo: PropTypes.bool,
```
Comment (mjoyce91) [inline src/Components/PositionTitle/PositionTitle.jsx:131]: 👍

**Comment by mjoyce91** [comment]
```
FAIL  src/Containers/Position/Position.test.jsx (10.806s)
 
  ● Position › can call the edit functions
 

 
    Warning: Failed prop type: The prop `location` is marked as required in `Position`, but its value is `undefined`.
 
        in Position (at Position.test.jsx:33)
 ```

File: src/Containers/Position/Position.jsx [hunk 4]
```diff
@@ -195,6 +209,12 @@ Position.defaultProps = {
   client: {},
   clientIsLoading: false,
   clientHasErrored: false,
+  location: {
+    pathname: '',
+    search: '',
+    hash: '',
+    key: '',
+  },
```
Comment (scott062) [inline src/Containers/Position/Position.jsx:217]: Test was failing because I had this defined with PropTypes.shape() instead of just an object

**Pull request merged**